### ISSUE TITLE
Lag dataset mal

### ISSUE THREAD

**Issue by ninsbl** [NINAnor/NINA_rapport_zotero#2]
For å gjør det lettere å sitere data som er brukt i analyser burde vi ha et mal / eksempel for det (også mtp. NINA stil).

Dataset fra Zenodo eller Pangea er lett å importere til Zotero og kommer som type Dokument.

Andre datalagre er litt mer vrient:

Dataset fra:
- GeoNetwork
- pyCSW
- GeoNode
- CKAN
- IPT

kan ikke importeres fra kontekst-menuen i nettleseren med Zotero plugin.

Så det bør gjøres noe andre steder også (dvs. ISP 19115/10139 import i Zotero, DublinCore RDF eksport i CKAN, pyCSW, GeoNetwork, bla.).

@frafra @mdlux: Maybe you have some ideas?
Hvis man har en DOI for datasettet i nettleseren importerer Zotero fint! Så det ville i utgangspunkt være tilstrekelig å tilby en DOI med datasettet...

**Comment by mdsnor** [comment]
Hey @ninsbl,
Our CKAN-based data portal will expose DOIs in its DCAT dataset metadata representation (RDF/XML serialization). Every dataset will list the DOI as identifier. Maybe it will be enough? Is it answering your question? 
However, I cannot point to a working example of it, for the moment.

**Comment by ninsbl** [comment]
Yes, a DOI is enough to import properly. Thanks. So that answers my question...

What about data in IPT? Will that be merged with / integrated in CKAN or will DOIs be added there as well (IPT seems to support that in principle)...

In principle any (also multiple) DOI on a website will be picked up and users can choose which DOI to resolve and import...

On the Zotero side, Geographic data could be handled better though (some additional fields would be nice...). But that is a different issue. 

So, conclusion is NINA solutions support reference management systems, while Kartverket does not yet...

FYI: I posted a question on Zotero mailing list regarding ISO XML import, so we will see how that turns out......

**Comment by mdsnor** [comment]
In the case of IPT, a DOI is assigned upon publication on GBIF. If we add a catalogue entry for that same dataset in NINA's catalog, and additional DOI will be generated (and it can be added as a reference to the GBIF record). 

As for the Geographic data, probably I don't get exactly what you mean. From my perspective, it's not an issue, because everything we will expose as geospatial data in the geodata portal, will be also referenced as a record in the CKAN catalog, and a DOI will be added to that (considered main) metadata record. And probably also added to the GeoNode in a second moment.

**Comment by mdsnor** [comment]
To be more clear: every dataset ready to be shared, no matter type or format, will be included in the catalog at least as a metadata record. Every metadata record will be assigned a DOI. Additionally, geospatiall data will be also shared in the geodata portal for improved functionalities (data as an OGC service). Zotero should get info from the main catalog only.

**Comment by ninsbl** [comment]
ISO import would be required for e.g.:
https://kartkatalog.geonorge.no/metadata/arter-av-nasjonal-forvaltningsinteresse/a8456aed-441a-40c4-831f-46bcbe4e6ff1
(there is no DOI).

And NINAs IPT offers no DOI either: https://ipt.nina.no/resource?r=arko_gel
If that is added through CKAN that should be enough.

Anyway useful to know that all NINA users should be directed to the CKAN repo and not IPT/geodata.nina.no...
